### PR TITLE
0.1.0 release docs

### PR DESCRIPTION
**Pull request by mkernohanbc**
This PR aggregates content updates to various docs pages in Storybook and on GitHub to go live alongside the component library v0.1.0 release.

### CODE REVIEW

File: packages/react-components/src/stories/Introduction.mdx [hunk 1]
```diff
@@ -15,9 +15,14 @@ export const link =
 
 <Meta title="Introduction" />
 
-# B.C. Design System React Components
+# B.C. Design System React Components v{packageJson.version}
```
Comment (ty2k) [inline packages/react-components/src/stories/Introduction.mdx:18]: This new chunk **should** always work but let's write it like this instead to make sure:

```js
{packageJson?.version ? `v${packageJson.version}` : null}
```

File: packages/react-components/src/stories/Introduction.mdx [hunk 2]
```diff
@@ -15,9 +15,14 @@ export const link =
...
 
-This is a UI workshop for React components included in the [B.C. Design System](https://gov.bc.ca/designsystem). This app provides:
+<Subtitle>
```
Comment (ty2k) [inline packages/react-components/src/stories/Introduction.mdx:20]: Looks good!

Storybook introduction page

**Review by ty2k — CHANGES_REQUESTED** [review]
Looks awesome! One small change request for the way we inject the package.json version.

File: packages/react-components/src/stories/Introduction.mdx (hunk 2 above)
Comment (mkernohanbc) [reply packages/react-components/src/stories/Introduction.mdx:20]: Fixed in [993b7e0](https://github.com/bcgov/design-system/pull/419/commits/993b7e0e7094d5ee9b3953408d80c4db85c5c09f)!

**Review by ty2k — APPROVED** [review]
🚀

**Pull request merged**